pkg/destroy/openstack/openstack.go

```diff
@@ -1822,3 +1832,68 @@ func validateCloud(opts *clientconfig.ClientOpts, logger logrus.FieldLogger) err
 
 	return networkextensions.Validate(availableExtensions)
 }
+
+func clearVIPsPorts(opts *clientconfig.ClientOpts, filter Filter, logger logrus.FieldLogger) (bool, error) {
```

Comment:
candid question: if the installer can't create ports with direct assignment of addresses when using slaac/stateless, why would we take care of their destruction?
I agree that's for user experience, but when I look at the code I'm wondering if that's really something we want to carry.

Alternative: when we document how to deploy in dual stack, add a note about destroy and the fact the api/ingress ports aren't automatically destroyed.

Reply:
it's not the actual destruction of the port, just the clean up of it. We need to add sg and fips. When cleaning up those resources from the Port we make sure it remains in the same state as it was provided. Also, it allows destroy command to finalize otherwise it would hang on sg destroy.

Reply:
> it's not the actual destruction of the port, just the clean up of it

oh I see now. **Maybe** rename the function to `cleanVIPsPorts` :rofl: 

> When cleaning up those resources from the Port we make sure it remains in the same state as it was provide

:+1:

All good then.